docs/release/trg-0/trg-9-03.md

```diff
@@ -0,0 +1,255 @@
+---
+title: TRG 9.03 - KIT Artifacts
+---
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 19-Feb-2024  | Initial contribution                   |
+
+This explains each artifact a KIT can or must follow - depending on the graduation stage (TRG 8.02). An artifact is textual content or a technical component that is part of a KIT (e.g. a Vision statement).
```

Comment:
In my opinion, you should structure it differently.

The kit overview page i linked/mentioned before, should explain to you how you write kits, how the stages work and how the process works.

The TRGs should be short and easy to follow and basically tells you all the requirements and why (see the other TRGs).

Good KIT TRGs would be 'tutorial section' why: because tutorial help people to follow description: a then b then c, best practies etc.

Reply:
This is something I have to discuss with @danielmiehle

Reply:
Sooo do you discuss it first and than we rework the PR?

Reply:
Jup - I think we did now an update on the small changes like spelling and so on but this I will discuss with Daniel and than we come back with an idea on how to proceed.

Reply:
Daniel and me discussed this again. We would like to publish the TRGs as they are and get some feedback from KIT owners. That's why we publish it as a draft. I hope that's fine :)

Reply:
I don't like it as it will lead to the risk that this TRG wlll get moved over but fine for me